x11-libs/gdk-pixbuf-loader-webp/gdk-pixbuf-loader-webp-0_p20191003.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
Invalid copyright, don't you use `repoman`?

Reply:
I made it in 2019...

Reply:
How should I understand this then?

> recreate merge request with x11-libs/gdk-pixbuf-loader-webp 0.0.1 (freshly released)

should this be closed in favor of a different PR?